modules/wipesBidAdapter.md

```diff
@@ -0,0 +1,28 @@
+# Overview
+
+```
+Module Name: WIPES Bidder Adapter
+Module Type: Bidder Adapter
+Maintainer: contact@3-shake.com
+```
+
+# Description
+
+Connect to WIPES for bids.
+Publishers needs to be set up and approved by WIPES team to enable this adapter.
+Please contact support@wipestream.com for further information.
+
+# Test Parameters
+```javascript
+var adUnits = [
```

Comment:
Would you mind specifying also the media type configuration ? I mean mediaType.context - instream, outstream, etc. 

This is not mandatory but it is appreciated to test the bidder.